orientdb/README.md

```diff
@@ -64,7 +69,29 @@ WARNING: Creating a new database will be done safely with multiple threads on a
 * ```orientdb.remote.storagetype``` - Storage type of the database on remote server
     * This is only required if using a ```remote:``` connection url
 
-## Known Issues
 
+## Plot results
+
+To plot workload results install gnuplot, then create a directory to collect the outputs
+```
+mkdir -p orientdb/outputs/orientdb
+```
+load the data
+```
+./bin/ycsb load orientdb -s -P workloads/workloadc 
+```
+and run the workload, e.g. `workloadc`, by following the naming convention for the file name `outputRun_<workload>.txt`:
+```
+./bin/ycsb run orientdb -s -P workloads/workloadc > orientdb/outputs/orientdb/outputRun_workloadc.txt
+```
+
+After a successful worload run, goto the `plots` directory `cd orientdb/plots` and run
+```
+./tool.sh "plot" "save" ../outputs/orientdb/
+```
+The diagrams will be in the `images` directory.
+
```

Comment:
this is a nice addition, but it will be confusing to folks who download the orientdb specific tarball because these directories won't be present.

you can check the tarball like this:

```
10335-busbey:YCSB busbey$ mvn -pl orientdb -am verify
...
10335-busbey:YCSB busbey$ tar tzf orientdb/target/ycsb-orientdb-binding-0.18.0-SNAPSHOT.tar.gz 
...
```

What if we pulled this tooling out of the orientdb module and generalized it for ycsb runs?

Reply:
sounds good, but it should stay on ODB level in this PR. We could put a more general support into a new PR. Wdyt?